dicomImport/matRad_scanDicomImportFolder.m

```diff
@@ -30,7 +30,9 @@
 % LICENSE file.
 %
 % %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-
+%% print current status of the import script
+fprintf('The following things are not implemented:\n');
+fprintf('not implemented - \n use automatically the right dose file (at the moment just list all)\n');
 %% get all files in search directory
```

Comment:
Updated version of this information. Actually matching is done automatically, since I had only one DICOM with multiple plans - and therefore multiple dose series - I'd recommend to double check matching.